app/handlers/sessions_callback.rb

```diff
@@ -39,11 +39,11 @@ def authorized?
   end
 
   def handle
-    # Get an authentication object for the incoming data, tracking if we have
+    # Get an authentication object for the incoming data, tracking if
     # the object didn't yet exist and we had to create it.
 
-    authentication_data = { provider: @auth_data[:provider],
-                            uid: @auth_data[:uid]}
+    authentication_data = { provider: @data.provider,
```

Comment:
Actually it's used in the `where` and `create` methods so I guess this assignment avoids 1 duplication.